test/jdk/java/nio/file/Files/ReadWriteString.java

```diff
@@ -271,6 +288,32 @@ public void testMalformedRead(Path path, byte[] data, Charset csWrite, Charset c
         }
     }
 
+    /**
+     * Verifies that IOException is thrown when reading a file containing
+     * illegal bytes
+     *
+     * @param data the data used for the test
+     * @param csRead the Charset to use for reading the file
+     * @param expected exception class
+     * @throws IOException when the Charset used for reading the file is incorrect
+     */
+    @Test(dataProvider = "illegalInputBytes")
+    public void testMalformedReadBytes(byte[] data, Charset csRead, Class<CharacterCodingException> expected)
+            throws IOException {
+        Path path = Files.createTempFile("illegalInputBytes", null);
+        path.toFile().deleteOnExit();
```

Comment:
Although this is consistent with the existing tests in this source file, I think it would be better if we change it to create the temporary files in the current directory and not delete it. Leaving the file behind is useful for diagnosing issues where there re test failures. We can do that in a separate issue if you'd like.

Reply:
Done.

Reply:
> Done.

Thanks, that will be helpful if the test fails.